react/features/calendar-sync/middleware.js

```diff
@@ -200,7 +199,10 @@ function _getURLFromEvent(event, knownDomains) {
                     = urlRegExp.exec(field) || schemeRegExp.exec(field))
                         !== null
             ) {
-                return matchArray[0];
+                const parsedURI = parseURIString(matchArray[0]);
+
+                return `${parsedURI.protocol}//${parsedURI.host}/
+                ${parsedURI.room}${parsedURI.search}${parsedURI.hash}`;
```

Comment:
That's not what I said, right? I said it was safe to break inside the `${}`: either right after `${` or right before `}`.